client/modules/driver/reducers/route.js

```diff
@@ -206,7 +207,7 @@ export const createSelectors = path => {
 
   return {
     isFetching: state => get(state, path).routeFetching,
-    hasError: state => get(state, path).routeError,
+    hasError: state => get(state, path).error,
```

Comment:
The line above should just be `fetching` too shouldn't it?

Reply:
Yes, thanks for spotting that one James.